modules/apps/29-fee/module.go

```diff
@@ -36,7 +36,7 @@ func (AppModuleBasic) Name() string {
 }
 
 // RegisterLegacyAminoCodec implements AppModuleBasic interface
-func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {}
+func (AppModuleBasic) RegisterLegacyAminoCodec(*codec.LegacyAmino) {}
```

Comment:
Is the linter special casing this? Would be nice to be consistent with the rest:

```suggestion
func (AppModuleBasic) RegisterLegacyAminoCodec(_ *codec.LegacyAmino) {}
```

Reply:
I'd actually like to keep the cdc, let me see if it can be happy with that.